docs/getting-started/configure-app-with-plugins.md

```diff
@@ -22,41 +22,35 @@ The following steps assume that you have
 [created a Backstage app](./index.md) and want to add an existing plugin
 to it.
 
-We are using the
-[CircleCI](https://github.com/CircleCI-Public/backstage-plugin/tree/main/plugins/circleci)
-plugin in this example, which is designed to show CI/CD pipeline information attached
-to an entity in the software catalog.
+You can find many wonderful plugins out there for Backstage including the [Community Plugins Repository](https://github.com/backstage/community-plugins) and the [Backstage Plugin Directory](https://backstage.io/plugins).
+
+Adding plugins to your Backstage app is generally a simple process, and ideally each plugin will come with its own documentation on how to install and configure it. The general steps usually include:
 
 1. Add the plugin's npm package to the repo:
 
    ```bash title="From your Backstage root directory"
-   yarn --cwd packages/app add @circleci/backstage-plugin
+   yarn --cwd packages/app add @<scope>/<plugin-name>
```

Comment:
could tech radar be a good candidate?

Reply:
Going to cook something up now with Tech Radar 🧑‍🍳

Reply:
Maybe this works? [abf43ac](https://github.com/backstage/backstage/pull/29316/commits/abf43ac58e57a5d779e3f8f5a8a17b460ed8b0b8)